docs/ko/docs/advanced/response-cookies.md

```diff
@@ -0,0 +1,52 @@
+# 응답 쿠키
+
+## `Response` 매개변수 사용하기
+
+*경로 작업 함수*에서 `Response` 타입의 매개변수를 선언할 수 있습니다.
```

Comment:
```suggestion
*경로 작동 함수*에서 `Response` 타입의 매개변수를 선언할 수 있습니다.
```
here is the reference for Path operation in korean terms
https://github.com/fastapi/fastapi/discussions/3167